components/eam/src/physics/cam/zm_conv.F90

```diff
@@ -99,7 +125,18 @@ module zm_conv
    
    integer  limcnv       ! top interface level limit for convection
 
-   real(r8) :: tp_fac = unset_r8  ! PMA tunes tpert 
+   real(r8) :: tp_fac = unset_r8  ! PMA tunes tpert
+   real(r8) :: auto_fac = unset_r8
+   real(r8) :: accr_fac = unset_r8
+   real(r8) :: micro_dcs= unset_r8
+
+!++ MCSP   
```

Comment:
I don't think these code "markers" are necessary and they contribute clutter to the code, so I would suggest removing them.

Reply:
Okay. It makes sense in many instances where it's redundant, so I've removed them where applicable.